### PR TITLE
create mailer

### PR DESCRIPTION
**Pull request by SetterVet**

### CODE REVIEW

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -39,4 +39,8 @@ def set_event
   def event_params
     params.require(:event).permit(:description, :start_date, :finish_date, :comments, :summary, :status)
   end
+
+  def send_email_create_event
+    EventMailer.with(event: @event).create_event.deliver_now
```
Comment (VasylYaremkiv) [inline app/controllers/events_controller.rb]: We need user who creates event(`Inviter`).

**Comment by SetterVet** [comment]
image

File: app/views/event_mailer/create_event.html.haml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+%h1
+  Create Event for you
```
Comment (rubycop) [inline app/views/event_mailer/create_event.html.haml]: Text: "You have upcoming event!"

File: spec/controllers/events_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+RSpec.describe EventsController, type: :controller do
+  describe 'POST events#create' do
+    let!(:admin) { FactoryBot.create(:user, role: 0) }
+    let!(:event) { FactoryBot.build(:event, user: admin) }
+
+    before(:each) do
+      login_user admin
+    end
+
+    it 'increment emails count' do
```
Comment (VasylYaremkiv) [inline spec/controllers/events_controller_spec.rb]: Use better name for test. What test check?

File: spec/rails_helper.rb [hunk 4]
```diff
@@ -1,8 +1,8 @@
 # This file is copied to spec/ when you run 'rails generate rspec:install'
 require 'simplecov'
 SimpleCov.start
-require 'login_helper'
-require 'phone_helper'
+require 'support/login_helper'
+require 'support/phone_helper'
```
Comment (VasylYaremkiv) [inline spec/rails_helper.rb]: Do not need  require `support/login_helper` and `support/phone_helper`. The following line requires all files on support dir:
```
 Dir[Rails.root.join('spec', 'support', '**', '*.rb')].sort.each { |f| require f } 
```

File: spec/mailers/event_mailer_spec.rb [hunk 5]
```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe EventMailer, type: :mailer do
+  let!(:admin) { FactoryBot.create(:user, role: 0) }
```
Comment (VasylYaremkiv) [inline spec/mailers/event_mailer_spec.rb]: Do not set role for user. Move it to factory

File: config/initializers/active_mailer.rb [hunk 6]
```diff
@@ -0,0 +1,29 @@
+Rails.application.configure do
+  if Rails.env.development?
+    config.action_mailer.default_url_options = { host: 'localhost:5000' }
+
+    config.action_mailer.delivery_method = :letter_opener
+    config.action_mailer.perform_deliveries = true
+
```
Comment (VasylYaremkiv) [inline config/initializers/active_mailer.rb]: Extra line

File: config/credentials.yml.enc [hunk 7]
```diff
@@ -1 +1 @@
-Z05XGrULv5yQLxvScpYpgnUIOjPAKc7ndE7x57eTx7OQrUQLnI4wSfTaHL2RXs+YyXf2QUkjyLTZyCrbOKAaUpEILnVFaChtpJWXA1A+a5QOTOfHQG7cNvFCRrlutTJzHreoyY4St5Yfb7cV3OuYfasMrEbYQKouYDu5RuQ/1uWwlfiGl1u7J4RCVSDBpRizkxbp5shs6P3BSLGCm+h9aBnTX9KIqSCeah6S3Xmb1fIgWmfjMOJjRmrXYJHmy3Ohie7GCoor51CyqlwNlJ/+hsmaqAcgEjM/oLYMsqyTnNRI7AO3Yay10WPX1KmQCYy76e5J+V1+snLWgl8CXrczO09Ix0xlZ1EQQDN80El2TQu5E1IvqphEjotMl46DQtOJbo1mdEMCY037eYmWtoOMH9rUB7ff6a9CEewUIi4jqcLmMYpLNfyUX7zA1daVHisDAN8Lt27Qqaa9rrP61ODp5RHF2+giKi+ub/52jiowutLrtlcjZCXjiuRRENTG3KHGO9+p59f053E=--3hatG3lRr168VyZQ--+8ae6OLTGWIsakMfysBhPg==
\ No newline at end of file
+quzVuqeWdigwRECv4eiOV0KxCBut/wjC2U3oZBqzeaRCxQMZnNAzKZHQyucub2DBML4LkWkoHHRhIK1oaoMzMUKW3HBdn3QAsy+L11kDhUnfq++TVZLqj38SDPgteFklqlAXo/aJ05OhrlpBeHqHJ5sjBOzKrbv8DHjuPkSnoG2VCXEbSztlVxUh0cXvudjU/FuM7cyNggPFZzYm/zadP7gcV7H3HGgVwuUYAw4xJIq0kUWboOUd6yO8lU4ShPOq8Q2l6drkl1pv22G+YGhcu8YolkOARS9sAHPx7Yk3RaYbHUx6FDjv0+t7iD11foGL3D9CjHtRVgcYgLm7yB80gN+K0NxuanghrNmqrD7IHTZjtlC0GTkbd4y+Ip+VdZsePWvq8JlVNmH+fETD3xaZT//j7hSkWnyua3to782lRFK8DneTe+nj78XsUfTzJxCRpm2r3LpgQ2A437jN0IwGyQfxVUJlePeK6gP444Tz2B0G9wCFWZ/N4rVajHl60ysKc7BHdy3iE8Q=--+mS+/BfXmS9n3NU+--37aH/HHYvH6QfbFqidU0Gw==
```
Comment (VasylYaremkiv) [inline config/credentials.yml.enc]: Why this file was changed?

**Comment by SetterVet** [comment]
@afurm @rubycop @VasylYaremkiv

**Comment by SetterVet** [comment]
fixed 
@VasylYaremkiv @afurm @rubycop

**Pull request merged**